tests/unit/amo/components/TestCollectionManager.js

```diff
@@ -738,6 +738,27 @@ describe(__filename, () => {
     expect(root.find(Notice)).toHaveLength(0);
   });
 
+  it('calls clearTimeout when unmounting and timeout is set', () => {
+    const clearTimeoutSpy = sinon.spy();
+    const root = render({ clearTimeout: clearTimeoutSpy });
+
+    // Simulates the setTimeout behavior in which timeout is set.
+    root.instance().timeout = 50;
+
+    root.unmount();
+
+    expect(clearTimeoutSpy.callCount).toEqual(1);
```

Comment:
We should use `sinon` for this kind of assertions, because `sinon` gives us a better output:

```diff
diff --git a/tests/unit/amo/components/TestCollectionManager.js b/tests/unit/amo/components/TestCollectionManager.js
index f23bd5b92..0d7fc6416 100644
--- a/tests/unit/amo/components/TestCollectionManager.js
+++ b/tests/unit/amo/components/TestCollectionManager.js
@@ -742,12 +742,13 @@ describe(__filename, () => {
     const clearTimeoutSpy = sinon.spy();
     const root = render({ clearTimeout: clearTimeoutSpy });
 
+    const timeoutID = 123;
     // Simulates the setTimeout behavior in which timeout is set.
-    root.instance().timeout = 50;
+    root.instance().timeout = timeoutID;
 
     root.unmount();
 
-    expect(clearTimeoutSpy.callCount).toEqual(1);
+    sinon.assert.calledWith(clearTimeoutSpy, timeoutID);
   });
 
   it('does not call clearTimeout when unmounting and there is no timeout set', () => {

```